### ISSUE TITLE
Use namespace manager for all variables in PSy layer

### ISSUE THREAD

**Issue by arporter** [stfc/PSyclone#251]
There are still a few variable names that are hardwired in PSyclone. These need to be replaced with names generated by the NamespaceManager to avoid potential clashes with the names of variables passed from the Algorithm layer.

**Comment by TeranIvy** [comment]
I think this is now redundant in the light of Sergi's PR #689 (replace NameSpaces with SymbolTables) so I am closing the issue. Please reopen if there is the need to do so.

**Comment by sergisiso** [comment]
I am afraid #689 will not fix this issue as I am not replacing 'hardwired' symbols. I reopened the issue and propose that we rename the issue to "Use SymbolTable for all variables in Psy layer" once #689 is merged.

**Comment by TeranIvy** [comment]
Sounds good to me. Is there anything in common with #312?

**Comment by arporter** [comment]
This issue has now been superseded by the work to move to using the PSyIR backend for all APIs (#1010). Therefore I shall close it.
